server/static/css/student.css

```diff
@@ -497,10 +503,10 @@ header .logo {
 /* Small devices (tablets, 768px and up) */
 @media (min-width: 768px) {
   .subcontent {
-    padding:1em 7em;
+    padding:0 5%;
```

Comment:
Using previous rule for subcontent: 
![image](https://cloud.githubusercontent.com/assets/882381/17954968/91584d3e-6a33-11e6-89f4-9675c459041f.png)

After:
![image](https://cloud.githubusercontent.com/assets/882381/17954959/790d2f2e-6a33-11e6-90a2-87b6b66b3dfd.png)